### ISSUE TITLE
OI build docs need illumos-closed package

### ISSUE THREAD

**Issue by rmustacc** [illumos/docs#60]
Right now to get the closed-bins on OI you need to install the illumos-closed package as well which is missing from the list of packages.

**Comment by jclulow** [comment]
Is that true?  The instructions currently say (for OI) to:

```
sudo pkg install build-essential \
    runtime/python-35 \
    developer/gcc-7
```

Looking at the dependencies of `build-essential`, it really seems like it should pull in the `illumos-closed` package:

```
$ pkg contents -r -m build-essential | grep closed
depend fmri=developer/illumos-closed type=require
```

**Comment by rmustacc** [comment]
It did not for a user on IRC that I was working through this with.

**Comment by mtelka** [comment]
> Looking at the dependencies of `build-essential`, it really seems like it should pull in the `illumos-closed` package:
> 
> ```
> $ pkg contents -r -m build-essential | grep closed
> depend fmri=developer/illumos-closed type=require
> ```

Yes, exactly.  And it is so since 2016: https://github.com/OpenIndiana/oi-userland/pull/2335
